MetaMask/metamask-extension
# What's New - Extension Full Size View Advanced Setting

## **Description**

- adds mention of Full-size View Setting in What's New popup modal
- supports searching for the setting
- note: image used is a reused image - `app/images/onboarding-pin-browser.svg` 

## **Related issues**

Fixes: #

## **Manual testing steps**

1. Open a new instance of MM to ensure the notifications are set to not viewed
2. Open application in pop-up or home.html to see new What's App notification

## **Screenshots/Recordings**

#### Expanded View
<img width="1201" alt="Screenshot 2023-12-21 at 8 03 22 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/1c9f9d96-26ed-4b96-a3dc-8ba8a0b17b8c">
<img width="1216" alt="Screenshot 2023-12-21 at 8 03 45 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/99ec4414-5f94-4783-bfdc-1c854339903b">

#### Pop-up View
<img width="496" alt="Screenshot 2023-12-21 at 8 02 59 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/d4bbf4e4-834e-465d-9cdc-98456d653e07">
<img width="497" alt="Screenshot 2023-12-21 at 8 04 36 AM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/4e322538-fe4a-4fc2-8d57-9b941291e285">


## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
